src/constants/messages/user.js

```diff
@@ -1,3 +1,11 @@
 export const userErrorMessages = Object.freeze({
+
+    EDIT_USER_PROFILE_SUCCESS: "Edit user profile success",
+    EDIT_USER_PROFILE_FAILED: "Edit user profile failed",
+    EDIT_USER_PROFILE_FAILED_USER_ALREADY_EXIST:
```

Comment:
Add that there is a user with such email to the name of the constant, and not only exists

PS. I know there is an identical misname in another file but it'll be fixed